sql/repl_failsafe.cc

```diff
@@ -191,6 +186,50 @@ static my_bool show_slave_hosts_callback(THD *thd, Protocol *protocol)
     }
     protocol->store((uint32) si->port);
     protocol->store(si->master_id);
+
+    if (gtid_state_from_binlog_pos(si->gtid_state_sent.log_file,
+                                   (uint32) si->gtid_state_sent.log_pos,
+                                   &gtid_sent, &err_msg))
+    {
+      gtid_sent.length(0);
+      DBUG_ASSERT(err_msg);
+      if (global_system_variables.log_warnings >= 2)
```

Comment:
I am sure about the `warning` level. `info` would be carrying lesser :-) incrimination.